gcloud/orb.yaml

```diff
@@ -264,11 +263,17 @@ jobs:
             cp "./<<parameters.packagename>>/requirements.txt" requirements.txt
             cp "./<<parameters.packagename>>/mains/<<parameters.funcname>>.py" main.py
             sed -i "s/\${GEMFURY_TOKEN}/$GEMFURY_TOKEN/g" requirements.txt
-            zip -r "<<parameters.funcname>>.zip" "./<<parameters.packagename>>"/* main.py requirements.txt
```

Comment:
Hmm one potential issue here is that our cloud functions are laid out such that there are other folders here -- without selectively only zipping up `"./<<parameters.packagename>>"/*`, we might be deploying more code than we mean to. I wonder if a staging folder would be a requirement here now?

Reply:
Yes, we're literally deploying all of our function code 😛

Reply:
But we were already doing that

Reply:
https://console.cloud.google.com/functions/details/us-central1/gcs_transcripts_deleter?project=talkiq-echelon&tab=source

Reply:
I'm not sure why this is the case, but you're right, we shouldn't be deploying all of our function code.